### ISSUE TITLE
Unexpected empty cell with a where condition

### ISSUE THREAD

**Issue by jbe456** [atoti/atoti#40]
## Steps to reproduce

```python
import pandas as pd

dummy_df = pd.DataFrame(
    data={
        "id": [0, 1, 2, 3],
        "Name": ["John", "John", "Thomas", "Mike"],
        "Score": [10, 8, 5, 12],
    }
)

import atoti

session = atoti.create_session()
dummy_store = session.read_pandas(
    dummy_df, keys=["id"], types={"id": atoti.types.STRING, "Score": atoti.types.DOUBLE}
)
cube = session.create_cube(dummy_store)

cube.measures["underlying"] = atoti.agg.single_value(
    atoti.where(cube.measures["Score.SUM"] == 12, 1000),
    scope=atoti.scope.origin(cube.levels["Name"]),
)

cube.measures["final"] = atoti.where(cube.measures["underlying"] == None, 'NOTHING', 'SOMETHING')
```

## Actual Result

The measure `final` returns an empty cell for `Total Name` although `underlying` has a value:

![image](https://user-images.githubusercontent.com/4415696/81863150-0835bc00-956b-11ea-95b4-34f32f421874.png)

## Expected Result

I would expect the final measure to return "SOMETHING" for "Total Name"

## Environment

- Atoti: 0.4.0.20200512194154
- Python: 3.8.2
- Operating System: linux

**Comment by fabiencelier** [comment]
This is related to the issue #17 and the comment about the where: single_value adds Name as a required level and where uses this required level to do a "per" which make it null when Name is not expressed

**Comment by fabiencelier** [comment]
My bad, I was wrong.
All the aggregation functions add required levels when using `scope`, but they are only transmit it to the next function in the case of single_value. If you do your use case with a `tt.agg.sum` instead of the single value you will have the expected result.

I will change the behavior of single value so it works as the other aggregation functions. It will only transmit the required levels if it is a store measure (`m["test"] = store["test"]`)

**Comment by fabiencelier** [comment]
Fixed in 0.4.0
